pkg/image/api/helper.go

```diff
@@ -38,18 +49,19 @@ func ParseDockerImageReference(spec string) (DockerImageReference, error) {
 	// TODO replace with docker version once docker/docker PR11109 is merged upstream
 	stream, tag, id := parseRepositoryTag(spec)
 
+	prefix, stream := splitProtocolPrefix(stream)
 	repoParts := strings.Split(stream, "/")
 	switch len(repoParts) {
 	case 2:
 		// namespace/name
-		ref.Namespace = repoParts[0]
+		ref.Namespace = prefix + repoParts[0]
```

Comment:
Is this to support `https://registry/busybox`? i.e. just the repo name, but no namespace specified?